.kokoro/verify-docker-images.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Script to verify the presence of GraalVM docker test images tagged with the latest java shared config version.
+
+javaSharedConfigVersion=1.5.7 # {x-version-update:java-shared-config:current}
+imageNames=( "graalvm_a" "graalvm_b")
```

Comment:
Perhaps we could avoid hardcoding this -- and instead parse the images from cloudbuild configuration?

```
cat cloudbuild.yml | grep "  - gcr.io/cloud-devrel-public-resources/" | cut -d "/" -f 3 | cut -d ":" -f 1
```

Reply:
That's a nice improvement, done.